docker/linux/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.8'
+
+services:
+  mapdl-container:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      - VERSION=${VERSION}
+      - USERNAME=mapdl
+      - USER_UID=1000
+      - LICENSE_SERVER=${LICENSE_SERVER}
+      - AWP_ROOT222=/ansys_inc
```

Comment:
Is this a variable linked to a unified install? Should it be `AWP_ROOT241`?

Reply:
it is.